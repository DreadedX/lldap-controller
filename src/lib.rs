pub mod context;
pub mod lldap;
pub mod resources;
