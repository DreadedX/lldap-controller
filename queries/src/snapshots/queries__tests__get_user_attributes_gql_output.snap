---
source: src/lib.rs
expression: operation.query
---
query GetUserAttributes {
  schema {
    userSchema {
      attributes {
        name
      }
    }
  }
}
